src/semantize/typing.nit

```diff
@@ -191,9 +195,9 @@ private class TypeVisitor
 		if sup == null then return null # Forward error
 
 		if sup == sub then
-			self.modelbuilder.warning(node, "useless-type-test", "Warning: expression is already a `{sup}`.")
+			if not disable_warning then self.modelbuilder.warning(node, "useless-type-test", "Warning: expression is already a `{sup}`.")
```

Comment:
It would be cleaner to

1. Add a local method `warn`
2. always call `warn`
3. let `warn` decide if it actually needs to write to stdout or not

Reply:
It's a better solution, thank you for the advice :+1: